pkg/samples/list.go

```diff
@@ -156,7 +156,7 @@ var List = map[string]*SampleData{
 	"multiple-plan-subscriptions":      multiPlanSubscriptions,
 	"set-up-subscriptions":             singleSubscription,
 	"web-elements-sepa-debit-payment":  sepaPayment,
-	"oxxo-payment": 					oxxoPayment,
+	"oxxo-payment":                     oxxoPayment,
```

Comment:
I honestly don't see what the diff between these two are, but I guess it now passes `make lint` 🤷‍♀ . I ended up doing gofmt and just copy/pasting the output into VSCode and saving it that way

Reply:
go likes it when everything is vertically lined up